src/parsec/agent/agentd.cpp

```diff
@@ -142,6 +143,16 @@ auto main(int argc, char** argv) -> int {
             broker,
             log,
             cfg.value());
+    } else if(cfg->m_runner_type == cbdc::parsec::runner_type::py) {
```

Comment:
(optional) I think this full-block might be a candidate for replacement with `switch`/`case` (so that the number of supported runners doesn't necessarily change the start-up time).

However, there's a deeper question that may be worth considering here: are runners a good candidate for pulling into a dynamically-linked library enabling the installation/support of new runners without recompilation? This is not a very small engineering effort; but something like [dylib](https://github.com/Layty/Dylib) might enable us to load the runners at runtime.

This might be a particularly positive way of pulling runners out of the main repo and modularizing them. Food for thought.

Reply:
WRT the first comment, I would agree that we ought to make this a switch statement -- for this, I think what'd be best is defining an enum with runner types 

WRT the second comment, I think that's actually a very interesting idea -- and i am definitely for pulling runners out of the main repository. This may be a task for a later PR but I think this is good to keep in mind for sure